cincoctrl/cincoctrl/templates/findingaids/list_records.html

```diff
@@ -54,22 +54,22 @@ <h2>Your collection guides</h2>
             {% if records_page.has_other_pages %}
               <div class="btn-group" role="group" aria-label="Record pagination">
                 {% if records_page.has_previous %}
-                  <a href="?page={{ records_page.previous_page_number }}" class="btn btn-outline-primary">&laquo;</a>
+                  <a href="{{ base_page_url }}page={{ records_page.previous_page_number }}" class="btn btn-outline-primary">&laquo;</a>
```

Comment:
Is there a reason not to use the [`url` template tag](https://docs.djangoproject.com/en/5.2/ref/templates/builtins/#url) and pass it the URL name? 
`<a href="{% url 'manage_records' page= records_page.previous_page_number %}"...`

Reply:
I'm trying to maintain the current filters applied from above.